.github/workflows/build-tokens.yml

```diff
@@ -26,12 +21,32 @@ jobs:
           node-version: "14"
       - name: Install Node Dependencies
         run: npm install
-      - name: Run token-transformer
+      - name: Pull any branch changes
```

Comment:
Doing this in both jobs since we've run into issues where the job errors out in the commit step since it's behind the origin branch

Reply:
Interesting, so this basically re-bases the branch with origin, as there was merging errors?

Reply:
Yeah, I ran into that issue in this run: https://github.com/Fullscript/aviary-tokens/runs/7115843713?check_suite_focus=true

So I figured just in case it happens again we could have this as a failsafe